gobblin-utility/src/test/java/gobblin/util/SchedulerUtilsTest.java

```diff
@@ -42,66 +42,72 @@
 @Test(groups = {"gobblin.util"})
 public class SchedulerUtilsTest {
 
-  private static final File JOB_CONF_ROOT_DIR = new File("gobblin-test/test-job-conf-dir");
-  private static final File SUB_DIR1 = new File(JOB_CONF_ROOT_DIR, "test1");
-  private static final File SUB_DIR11 = new File(SUB_DIR1, "test11");
-  private static final File SUB_DIR2 = new File(JOB_CONF_ROOT_DIR, "test2");
+  private File jobConfigDir;
+  private File subDir1;
+  private File subDir11;
+  private File subDir2;
 
   @BeforeClass
   public void setUp()
       throws IOException {
-    SUB_DIR1.mkdirs();
-    SUB_DIR11.mkdirs();
-    SUB_DIR2.mkdirs();
+    this.jobConfigDir = java.nio.file.Files.createTempDirectory("gobblin-test_job-conf").toFile();
```

Comment:
I can't add `java.nio.file.Files` because a Google class named `Files` is already being imported and used.